web/crux-ui/src/pages/products/[productId]/versions/[versionId]/deployments/[deploymentId].tsx

```diff
@@ -120,12 +122,16 @@ const DeploymentDetailsPage = (props: DeploymentDetailsPageProps) => {
   const onOpenLog = () => router.push(deploymentDeployUrl(product.id, version.id, deployment.id))
 
   const onDeploy = () => {
-    const error = getValidationError(deploymentSchema, deployment)
-    if (error) {
-      console.error(error)
-      toast.error(t('errors:invalid'))
-      return
-    }
+    deployment.instances.map(instances => {
```

Comment:
this way we are going to toast the user per instance. we should toast only once

i recommend .reduce instead of map